model.py

```diff
@@ -63,6 +63,9 @@ def is_supported(self, config):
             if not self.is_supported_mode(config):
                 return False, ("casual" if self.rated else "rated")
 
+            if self.challenger_name in config.get("block_list", []):
+                return False, "blocked"
```

Comment:
The message `"blocked"` is not a valid reason for declining a challenge. See the list of possible reasons in the [Lichess API documentation](https://lichess.org/api#tag/Challenges/operation/challengeDecline). The translation of each reason value can be found in [this file from the lila repository](https://github.com/lichess-org/lila/blob/b0ab35f98e3b535fe93ac653140d77abfa950a65/translation/source/challenge.xml).